test/spmd/tensor/test_experimental_ops.py

```diff
@@ -0,0 +1,60 @@
+from torch.testing._internal.distributed._tensor.common_dtensor import (
+    DTensorTestBase,
+    DEVICE_TYPE,
+)
+import torch
+
+from torch.distributed._tensor import DeviceMesh, distribute_tensor
+from torch.distributed._tensor.placement_types import (
+    Replicate,
+    Shard,
+    _Partial,
+)
+from spmd.tensor.experimental_ops import *  # noqa: F401 F403
+
+
+class TraceModuleTest(DTensorTestBase):
+    @property
+    def world_size(self):
+        return 2
+
+    def test_native_layernorm(self):
+        self.init_pg(backend="nccl" if DEVICE_TYPE == "cuda" else "gloo")
```

Comment:
setting up pg is actually what `@with_comms` is doing, maybe consider using that?